Twilio-org/phonebank
# Fix skip call error

## Description
- Removed unnecessary firing of assignCall action on client to avoid multiple call assignments. -
- Updated service and test to catch the case where a user might have multiple calls assigned. Thoughts on this part especially welcome.
- Update all call services to make sure calls are updating timestamps.

### Test
- Log in and start campaigning
- Notice user will have only one call assigned at once
- Try skipping through contacts to see the feature work
